src/jv_aux.c

```diff
@@ -427,6 +427,70 @@ jv jv_setpath(jv root, jv path, jv value) {
   return jv_set(root, pathcurr, jv_setpath(subroot, pathrest, value));
 }
 
+jv jv_addpath(jv root, jv path, jv add){
```

Comment:
Can we have a comment on what this is supposed to do?  Is it different from `jv_setpath()`?